python/setup.py

```diff
@@ -558,6 +558,11 @@ def pip_run(build_ext):
     copied_files = 0
     for filename in setup_spec.files_to_include:
         copied_files += copy_file(build_ext.build_lib, filename, ROOT_DIR)
+    if sys.platform == 'win32':
+        print('copy_file uses', build_ext.build_lib, ROOT_DIR)
+        shutil.copy(r'c:\Windows\system32\msvcp140.dll',
+                        os.path.join(build_ext.build_lib, 'ray'))
```

Comment:
```suggestion
        print("copy_file uses", build_ext.build_lib, ROOT_DIR)
        shutil.copy(r"c:\Windows\system32\msvcp140.dll",
                        os.path.join(build_ext.build_lib, "ray"))
```

Reply:
We use double quotes for literals

Reply:
thanks, fixed and removed the debug print